### ISSUE TITLE
When will the SMP of raspberry Pi 4 come true？Or there is some document for the Implementation of SMP？

### ISSUE THREAD

**Issue by huyu168** [RT-Thread/rt-thread#4623]
i tried to make the firmware for raspi 4 , but the smp seems to go wrong， the log is below：
`libcpu\arm\cortex-a/start_gcc.S:425: undefined reference to `secondary_cpu_c_start'`
it seems there is no code related to SMP for the BSP.

**Comment by BernardXiong** [comment]
Yes. The raspberry pi did not support SMP right now. Welcome to submit your PR and you can refer to qemu-vexpress-a9 bsp.
